NIPTool/models/converters.py

```diff
@@ -0,0 +1,36 @@
+def empty_str(x):
+    """Convert empty string to None"""
+
+    if not isinstance(x, str):
+        return x
+    x = x.strip()
+    if not x:
+        return None
+    return x
+
+
+CONVERTERS = {
+    str: ["SampleProject"],
```

Comment:
The `str` is a reserved keyword in python so this should not be used. Its the function that converts something to a string like `str(2)`

Reply:
Ill not use this code, but Im still intrested in this..! Here the function str, is being used as a key in the dict. the function str is not being replaced by anything. Would your suggestion be to make a make_str() function that will just call str(), and then use make_str as key int the dict?

Reply:
I think I would prefer to use a function but I do not entirely understand the situation. Sounds more explicit to have a function that converts stuff to a string